styles.css

```diff
@@ -78,3 +79,123 @@ ul {
 .connect {
   text-decoration: none;
 }
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 20px;
+  row-gap: 88px;
+  margin-left: 10px;
+}
+
+.works h2 {
+  margin-block-start: initial;
+  margin-block-end: initial;
+  font-weight: 700;
+  font-size: 32px;
+  margin-top: 10px;
+}
+
+.works p {
+  margin-block-start: initial;
+  margin-block-end: initial;
+  font-weight: 400;
+  size: 15px;
+  width: 295px;
+}
+
+.u-c {
+  display: flex;
+  justify-content: space-between;
+  width: 270px;
+  list-style: none;
+}
+
+.u-html {
+  display: flex;
+  list-style: none;
+}
+
+.u-html li {
+  margin-right: 15px;
+  background: #8f98eaa7;
+  padding: 6px;
+  border-radius: 8px;
+  font-weight: 500;
+}
+
+h4 {
+  padding: 10px;
+  display: inline-block;
+  border: 3px solid #2c3cc8a7;
+  border-radius: 8px;
+  font-weight: 500;
+}
+
+.card1 {
+  flex-direction: column;
+  display: flex;
+  background: white;
+  padding: 16px;
+  border-radius: 4%;
+}
+
+.card2 {
+  flex-direction: column;
+  display: flex;
+  background: white;
+  padding: 16px;
+  border-radius: 4%;
+}
+
+.card3 {
+  flex-direction: column;
+  display: flex;
+  background: white;
+  padding: 16px;
+  border-radius: 4%;
+}
+
+.card4 {
+  flex-direction: column;
+  display: flex;
+  background: white;
+  padding: 16px;
+  border-radius: 4%;
+}
+
+.about-section {
+  padding: 16px;
+  background: white;
+  border-radius: 0 60px;
+}
```

Comment:
- [ ] the content inside this section should be  aligned with flexbox , kindly try to add it

Reply:
Thanks